frontend/resourceadm/utils/resourceUtils/resourceUtils.ts

```diff
@@ -24,31 +32,30 @@ export const getResourcePageTextfieldError = (text: SupportedLanguageKey<string>
  *
  * @returns the string to display
  */
-export const convertResourceTypeToDisplayString = (resourceType: ResourceTypeOption) => {
-  if (resourceType === 'Default') return 'Standard';
-  else if (resourceType === 'Systemresource') return 'System ressurs';
-  else if (resourceType === 'MaskinportenSchema') return 'Maskinporten skjema';
-  return undefined;
+export const convertResourceTypeToDisplayString = (resourceType: ResourceTypeOption): string => {
+  return resourceTypeMap[resourceType];
 };
 
 /**
  * Maps the language key to the text
  */
-export const mapLanguageKeyToLanguageText = (val: 'nb' | 'nn' | 'en') => {
-  if (val === 'nb') return 'Bokmål';
-  if (val === 'nn') return 'Nynorsk';
-  return 'Engelsk';
+export const mapLanguageKeyToLanguageText = (val: 'nb' | 'nn' | 'en', translationFunction: any) => {
+  if (val === 'nb') return translationFunction('language.nb');
+  if (val === 'nn') return translationFunction('language.nn');
+  return translationFunction('language.en');
 };
 
 /**
  * Gets the correct text to display for input fields with missing value
  *
  * @param language the value
  * @param usageString the type of the field
+ * @param translationFunction the translation function
  */
 export const getMissingInputLanguageString = (
   language: SupportedLanguage,
-  usageString: string
+  usageString: string,
+  translationFunction: any
```

Comment:
Same as prev comment. :)
```suggestion
  translationFunction: (key: string, params?: object) => string
```